docs/en/Configuring-Alluxio-with-Swift.md

```diff
@@ -91,6 +91,14 @@ In case of failures, logs located under `tests/target/logs`. You may also activa
 
 # Swift Access Control
 
-The Swift crendentials specified in Alluxio (`fs.swift.user`, `fs.swift.tenant` and `fs.swift.password`) represents a Swift user. Swift service backend checks the user permission to the container. When Alluxio security is enabled, Alluxio loads the container ACL to Alluxio permission on the first time when the metadata is loaded to Alluxio namespace. It checks the container READ/WRITE ACL to see if the Swift user or "*" is in the access control list. If necessary, you can customize the Swift container ACL checking in `SwiftUnderFileSystem.java`. Swift user permission to the container is mapped to Alluxio permission to mounted directories and files.
+The Swift crendentials specified in Alluxio (`fs.swift.user`, `fs.swift.tenant` and `fs.swift.password`) represents a Swift user. Swift service backend checks the user permission to the container.
+If the given Swift user does not have the right access permission to the specified container, a permission denied error will be thrown.
+When Alluxio security is enabled, Alluxio loads the container ACL to Alluxio permission on the first time when the metadata is loaded to Alluxio namespace.
+It checks the container READ/WRITE ACL to see if the Swift user or "*" is in the access control list. If necessary, you can customize the Swift container ACL checking in `SwiftUnderFileSystem.java`.
```

Comment:
I wouldn't really include the last portion, since that requires source code editing.